server/Dockerfile

```diff
@@ -18,6 +18,15 @@ ENV DISTRIBUTION_URL https://repo1.maven.org/maven2/org/infinispan/server/infini
 RUN INFINISPAN_SHA=$(curl $DISTRIBUTION_URL.sha1); curl -o /tmp/server.zip $DISTRIBUTION_URL && sha1sum /tmp/server.zip | grep $INFINISPAN_SHA \
     && unzip -q /tmp/server.zip -d $HOME && mv $HOME/infinispan-server-* $HOME/infinispan-server && rm /tmp/server.zip
 
+# For Openshift
+USER root
+
+RUN chgrp -R 0 /opt/jboss/infinispan-server/standalone
+
+RUN chmod -R g+rw /opt/jboss/infinispan-server/standalone
+
+USER jboss
```

Comment:
Please do not integrate yet, I found an issue with this PR: it prevents accessing the CLI locally due to the arbitrary user id set at runtime.